client/client.go

```diff
@@ -368,22 +368,23 @@ func ParseHostURL(host string) (*url.URL, error) {
 	if len(protoAddrParts) == 1 {
 		return nil, fmt.Errorf("unable to parse docker host `%s`", host)
 	}
-
-	var basePath string
-	proto, addr := protoAddrParts[0], protoAddrParts[1]
-	if proto == "tcp" {
+	if proto, addr := protoAddrParts[0], protoAddrParts[1] ; proto == "tcp" {
```

Comment:
Not sure this is more readable than the original; the only different would be that this scopes the variables to the `if`, but that won't make much of a difference.